plugins/awssagemaker/flytekitplugins/awssagemaker/training.py

```diff
@@ -7,12 +7,17 @@
 from google.protobuf.json_format import MessageToDict
 
 import flytekit
-from flytekit.annotated.base_task import IgnoreOutputs, PythonTask, kwtypes
-from flytekit.annotated.context_manager import ExecutionState, FlyteContext, SerializationSettings
-from flytekit.annotated.interface import Interface
-from flytekit.annotated.python_function_task import PythonFunctionTask
-from flytekit.annotated.task import TaskPlugins
+from flytekit import FlyteContext, PythonFunctionTask
 from flytekit.common.tasks.sdk_runnable import ExecutionParameters
```

Comment:
I dont like this part that ExecutionParameters is coming from sdk_runnable. can we change that as well?